internal/transactionsubmission/monitor/monitor_service.go

```diff
@@ -0,0 +1,123 @@
+package monitor
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stellar/go/support/log"
+
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/monitor"
```

Comment:
Can we call it sdpMonitor, to differentiate it from the name of the current package?

```suggestion
	sdpMonitor "github.com/stellar/stellar-disbursement-platform-backend/internal/monitor"
```